src/Components/BackButton.js

```diff
@@ -0,0 +1,18 @@
+import IconButton from "@mui/material/IconButton";
+import { CaretLeft } from "phosphor-react";
+import { useNavigate } from "react-router-dom";
+
+export default function BackButton({ sx }) {
+  const navigate = useNavigate();
+
+  const onClick = (e) => {
+    navigate(-1);
+    e.stopPropogation();
```

Comment:
After misspelling this myself countless times, I now know it's e.stopProp**a**gation(), lol.

I'll make this change before merging the branch.

Reply:
Oof, my bad! Thanks for the fix